stdlib/3.9/zoneinfo/__init__.pyi

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import os
+import typing
+from datetime import datetime, tzinfo
+from typing import IO, Optional, Sequence, Type, Union
+
+class ZoneInfo(tzinfo):
+    _T = typing.TypeVar("_T", bound="ZoneInfo")
+    key: str
```

Comment:
```suggestion
    @property
    def key(self) -> str: ...
```

To match the implementation and mark this as readonly.

Reply:
I'm fine with this change, but that's only the pure Python implementation — the C implementation (which most people will see), will *not* be a property. I assume it won't matter, though, presumably anyone doing anything that *relies* on `.key` being a property will likely be doing stuff that is likely to make static analysis not that useful anyway?

Reply:
I see. Personally, I'd go the stricter (readonly) route so that if the Python implementation is used, code will still work, but I leave the decision to you.

Reply:
It's read-only in both C and pure Python, it's just not implemented as a property in the C extension (C extensions have native support for read-only attributes). I think the only difference would be some particularly funky stuff like if you try to call `zoneinfo.ZoneInfo.key.setter` or something of that nature. Nothing supported, really.